include/zephyr/drivers/smbus.h

```diff
@@ -46,7 +46,7 @@ extern "C" {
 #define SMBUS_CMD_QUICK			0b000
 
 /**
- * SMBus Byte protocol can send or receive one byte of data.
+ * SMBus Byte command can send or receive one byte of data.
```

Comment:
Please do not change protocol to command, it is written as specified here: http://smbus.org/specs/SMBus_3_1_20180319.pdf
and also to differentiate from `command` which is part of protocol packet.

Reply:
Sorry!